kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -289,7 +318,7 @@
             username: this.username,
             password: this.password,
             facility: this.facility,
-          });
+          }).catch(this.setFacilityModal());
```

Comment:
I don't entirely understand what's happening here. Some code comments?

Reply:
Explained a bit in https://github.com/learningequality/kolibri/issues/2552#issuecomment-368105613

Going for the behavior described in 
https://github.com/learningequality/kolibri/issues/2552#issuecomment-368140529
and
https://github.com/learningequality/kolibri/issues/2552#issuecomment-368141194

instead